python/ray/autoscaler/updater.py

```diff
@@ -246,9 +256,9 @@ def run(self,
                             "{} -> localhost:{}".format(local, remote))
                 ssh += ["-L", "{}:localhost:{}".format(remote, local)]
 
-        final_cmd = ssh + self.get_default_ssh_options(timeout) + [
-            "{}@{}".format(self.ssh_user, self.ssh_ip)
-        ]
+        final_cmd = ssh + ssh_options.get_ssh_options(
+            self.ssh_private_key,
+            timeout) + ["{}@{}".format(self.ssh_user, self.ssh_ip)]
```

Comment:
Should this be moved into a method on SSHOptions too?

Reply:
I don't think the user or IP changes through the lifetime of the updater so you could just pass that to the SSHOptions constructor

Reply:
I'll move the `ssh_private_key` into SSH Options, but I'll keep the `ssh_user` and `ssh_ip` out because they are used for `rsync_up`  (`<source> "{}@{}:<target>"`) and `rsync_down`  (` "{}@{}:<source>" <target>`). I would rather not move this logic out.